src/serverdlg.cpp

```diff
@@ -593,13 +593,13 @@ void CServerDlg::OnClearRecordingDirClicked()
 
 void CServerDlg::OnSysTrayActivated ( QSystemTrayIcon::ActivationReason ActReason )
 {
-    #ifdef _WIN32
+#ifdef _WIN32
```

Comment:
Attention on this one ;-)

Reply:
Hm....noticed this came in my PR after merging master into it. Guess I should somehow be able to "remove" this from the PR.

Reply:
Is there a method I can use to remove this from my branch? I.e. using git revert perhaps?

Reply:
git rebase probably. But I‘m not an expert with that unfortunately.

Reply:
It got included when I did the rebase to master....going to test/try...worst case I'll have to create a new PR.